Sources/PrimerSDK/Classes/Core/Logging/PrimerLogger.swift

```diff
@@ -7,7 +7,7 @@ public struct PrimerLogMetadata {
     public let file: String
     public let line: Int
     public let function: String
-
+    
```

Comment:
Where are these spaces coming from? Is this something we can catch with 🥁 SwiftLint? 🥳